CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ Remember to align the itemized text with the first line of an item within a list
 * New Functionality
   * Added {func}`jax.numpy.trapezoid`, following the addition of this function in
     NumPy 2.0.
+  * The {func}`jax.experimental.array_api.clip` function has been added in
+    compliance of the array API 2023 standard.
```

Comment:
We typically don't mention `experimental` changes in the changelog, but let's merge and fix later.